docs-ref-services/postgresql-mysql.md

```diff
@@ -54,46 +54,50 @@ Please see the [Python SDK authentication](https://docs.microsoft.com/python/azu
 
 #### Example
 
-Create a MySQL 5.7 Database resource and restrict access to a range of IP addresses using a firewall rule.
+Creates an Azure Database for MySQL server and restricts access to a range of IP addresses using a firewall rule.
 
 ```python
```

Comment:
Includes steps on what the user needs to do to execute this script.  i.e. copy and paste this into a .py file and execute with file.py, etc.

Reply:
@rloutlaw - do you have recommendations? I don't think any of the equivalent docs for other services show steps?